man/systemd.resource-control.xml

```diff
@@ -691,6 +691,26 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowBindPorts=<replaceable>IP_PORT</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow binding a socket address only to certain IP ports specified by <replaceable>IP_PORT</replaceable>.
+          Same ports will be allowed for both IPv4 and IPv6 protocols. Port value must be in [0, 65535] range.
```

Comment:
we usually write integer ranges in the docs as 0…65535

hmm, `0` is not actually allowed, is it? that's not a valid ip port after all